pytest_ordering/__init__.py

```diff
@@ -35,72 +36,44 @@ def pytest_configure(config):
 
 
 def pytest_collection_modifyitems(session, config, items):
-    items[:] = list(_order_tests(items))
+    grouped_items = {}
 
+    for item in items:
 
-def orderable(marker_name, marker_info):
-    if not hasattr(marker_info, 'kwargs'):
-        return False
-    elif 'order' in marker_info.kwargs:
-        return True
-    else:
-        match = re.match('^order(\d+)$', marker_name)
-        return bool(match) or marker_name in replacements
+        for mark_name, order in orders_map.items():
```

Comment:
What do you think the behavior should be if there are multiple order markers on a test? I think this implementation will make it indeterminate which is picked. I think I'd prefer acting like there is no ordering set, and perhaps adding an erroring test with a message about how you can't have multiple ordering markers. Or, maybe there is some other behavior that makes sense? What do you think?

I don't mind leaving this as-is for now.